task3/src/routers/controllers/user.controller.ts

```diff
@@ -0,0 +1,96 @@
+import { Request, Response } from 'express';
+import { Op } from 'sequelize';
+
+import UserModel from '../../models/user.model';
+
+import { DEFAULT_USERS } from '../../data/data-users';
+
+class UserController {
+    constructor() {
+        UserModel
+            .sync({ force: true })
+            .then(() => UserModel.bulkCreate(DEFAULT_USERS));
+    }
+
+    async findById (req: Request, res: Response) {
+        const { id } = req.params;
+    
+        try {
+            const user = await UserModel.findByPk(id);
+
+            if (!user) {
+                res.status(404).json({
+                    error: 'User doesn\'t exists'
+                });
+            } else {
+                res.status(200).json({ user });
+            }
+        } catch (error) {
+            res.status(404).json({ error });
```

Comment:
In case of error during db request the 404 status may confuse. 500 (Internal Service Error) will be more applicable. Also, check how to make general error handler with middleware

Reply:
Fixed